packages/api/cms-api/src/user-permissions/decorators/affected-entity.decorator.ts

```diff
@@ -0,0 +1,16 @@
+import { EntityClass } from "@mikro-orm/core";
+import { CustomDecorator, SetMetadata } from "@nestjs/common";
+
+type AffectedEntityOptions = {
+    argsSelector?: string;
+    pageTreeArgsSelector?: string;
```

Comment:
This should be pageTreeNodeArgsSelector to show that it is to retrieve the respective node

Reply:
https://github.com/vivid-planet/comet/pull/1223/commits/117473f9f23fa0b3f9366b9886f2efc570fc0eaf